src/widgets/section/Subsection.tsx

```diff
@@ -0,0 +1,11 @@
+import WidgetBase from '@dojo/framework/widget-core/WidgetBase';
+import { WidgetProperties } from '@dojo/framework/widget-core/interfaces';
+import { tsx } from '@dojo/framework/widget-core/tsx';
+
+import * as css from './Subsection.m.css';
+
+export default class Subsection extends WidgetBase<WidgetProperties> {
```

Comment:
A standalone `Subsection` widget was created so this styling (and potential functionality in the future) could be used outside of the content pipeline.